services/server/src/sourcify-chains-default.json

```diff
@@ -126,6 +126,10 @@
       }
     }
   },
+  "7078815900": {
+    "sourcifyName": "Ethereum Mekong Testnet",
```

Comment:
Maybe we can add the blockscout API:

https://explorer.mekong.ethpandaops.io/

Reply:
Nice! I wasn't aware of it, it wasn't listed in https://mekong.ethpandaops.io/